.dev/assets/shared/css/header/header.scss

```diff
@@ -30,6 +30,24 @@
 	}
 }
 
+body.has-sticky-header {
+
+	.site-header {
+		position: sticky;
+		top: 0;
+
+		&:not(.has-background) {
+			background: var(--go--color--background,0,0,100%);
```

Comment:
Is this passing the stylelint? I would assume we need spacing after comma like : `var(--go--color--background, 0, 0, 100%)`